README.md

```diff
@@ -10,4 +10,396 @@ TODO: description
+- `specialty` _{string}_ - The medical contact's specialty
+- `phone_number` _{string}_ - The medical contact's phone number
+- `notes` _{string}_ - Notes about the medical contact
+
+**Returns**
+
+- A success message
+- An object with the edited medical contact
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the first_name, last_name, or phone_number is empty or a stream of empty spaces
+
+#### `DELETE /api/medicalContacts/:medicalContactId?` - Delete a medical contact
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `403` if the user is not logged in
+- `403` if the user is not the owner of the medical contacts
+- `404` if the medicalContactId is invalid
+
+### Medication
+
+#### `GET /api/medications?owner=USERNAME` - Get medications by owner
+
+**Returns**
+
+- An array of medications created by user with username `owner` in alphabetical order with respect to the name of the medication
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
+
+#### `GET /api/medications?name=NAME` - Get medications by name
+
+**Returns**
+
+- An array of medications with name `NAME` in alphabetical order with respect to the name of the medication
+
+**Throws**
+
+- `400` if `NAME` is not given
+
+#### `POST /api/medications` - Create a new medication
+
+**Body**
+
+- `active` _{string}_ - Whether the medication is in use or not
+- `name` _{string}_ - The medication's name
+- `generic_name` _{string}_ - The medication's generic compound name
+- `dose` _{string}_ - The medication's dose
+- `notes` _{string}_ - Notes about the medication
+
+**Returns**
+
+- A success message
+- An object with the created medication
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the name, generic_name, or dose is empty or a stream of empty spaces
+
+#### `PATCH /api/medications/:medicationId?` - Edit the information of a medication
+
+**Body**
+
+- `active` _{string}_ - Whether the medication is in use or not
+- `name` _{string}_ - The medication's name
+- `generic_name` _{string}_ - The medication's generic compound name
+- `dose` _{string}_ - The medication's dose
+- `notes` _{string}_ - Notes about the medication
+
+**Returns**
+
+- A success message
+- An object with the edited medication
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the name, generic_name, or dose is empty or a stream of empty spaces
+
+#### `DELETE /api/medications/:medicationId?` - Delete a medication
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `403` if the user is not logged in
+- `403` if the user is not the owner of the medication
+- `404` if the medicationId is invalid
+
+### Entry
+
+TODO: Linda
+
+
+### Trend
+
+#### `GET /api/trends` - Get all trends for a given user  
+**Returns**  
+- An array of trends created by user with given username (taken from session)  
+**Throws**  
+- `400` if `user` is not found in session  
```

Comment:
I think this should be [401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) instead of 400